globals.go

```diff
@@ -213,9 +213,29 @@ const (
 	// app server should log to stdout.
 	standardOutputLoggingOverride = "STDOUT"
 
-	DefaultTaskActivator   = ""
-	StepbackTaskActivator  = "stepback"
+	// DefaultTaskActivator is a deprecated legacy activator that used
+	// to be a majority of non-stepback and non-API activations.
+	DefaultTaskActivator = ""
```

Comment:
can we change the name of this to LegacyTaskActivator so new code knows not to use it?